src/test/java/cz/cvut/kbss/termit/service/business/ResourceServiceTest.java

```diff
@@ -351,12 +352,12 @@ void getFilesThrowsUnsupportedAssetOperationExceptionWhenSpecifiedResourceIsNotD
     }
 
     @Test
-    void addFileToDocumentPersistsFileAndUpdatesDocumentWithAddedFile() {
+    void addFileToDocumentPersistsFileAndPersistsDocumentWithAddedFile() {
```

Comment:
These are now IMHO two different cases: 1. File added to an existing document, 2. File created along with the document. Thus, they should be covered by two tests.

Reply:
You are absolutely right.

Unfortunately, I have to postpone this bit, need to do other stuff now.